study.md

```diff
@@ -44,21 +44,21 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+View-state is the method that the page framework uses to preserve page and controlled values.  There are multiple different views that exist on the same page meaning it doesn't need to refresh.
 ```
 
 ## Define the advantages of an SPA over a traditional application
 
 What are the advantages of a single page application over a traditional appliction?
 
 ```md
-<!-- your answer here -->
+More fluid user experience as the page doesn't refresh.  This is primarily it.  If thsi is your main priority then SPAs may be the may to go but if content is the priority then maybe not.  The page also tends to load slower initially and can consure your browser's navigation.
 ```
 
 ## List some technologies used to make SPA's
 
 What are some of the tools used to create single page applications?
 
 ```md
-<!-- your answer here -->
+The client requests the HTML from the server initially.  Any other requests are made with AJAX and the server returns JSON.
 ```
```

Comment:
This is a great description of the process. Every day there are more tools including JavaScript frameworks (Ember, React, Angular) to help us make single page apps!!